src/js/blue-button/containers/DownloadPage.jsx

```diff
@@ -9,10 +11,7 @@ export class DownloadPage extends React.Component {
       pdfFileSize: '124 KB',
```

Comment:
Will this number be available when the component mounts or only after the user clicks 'Download PDF'?